data_structures/singly_linked_list.js

```diff
@@ -12,21 +12,29 @@
  */
 
 class Node {
-    constructor(){
-        
+    constructor(value){
+        this.value = value
+        this.next = null
     }
     
     //returns the node's value
-    get_value(){}
+    get_value(){
+        return this.value
+    }
     
     //returns the node's next node
-    get_next(){}
+    get_next(){
+        return this.next
+    }
     
     //set the node's next node pointer to the passed in node
-    set_next(node){
+    set_next(value){
```

Comment:
This should be a node being passed in not a value. So the function should look like
```
set_next(node){
    this.next = node
}
```